install/Linux.shtml

```diff
@@ -145,14 +145,32 @@
 
     </div>
 
-    <h4>Desktop Icons</h4>
-      <p>Desktop icons are not required.  PanelPro and DecoderPro can be started by clicking on the
-      scripts within the JMRI install in the file manager or from the command line.  For example:
+    <h4>Application Icons</h4>
+      <p>Application icons are not required.  PanelPro and DecoderPro can be started by clicking on
+      the scripts within the JMRI install in the file manager or from the command line.  For example:
       <code>~/JMRI/PanelPro</code></p>
 
+      <p>As of the <strong><em>JMRI 5.3.6</em></strong> test release, the <strong>CreateJmriApps.sh
+      </strong> shell script is included in Linux JMRI install.  The script creates Linux application
+      definitions for DecoderPro and PanelPro.  These are used by Linux application launchers such
+      as the LM popup in Mint, the Raspberry popup in Raspberry Pi OS, the application grids, etc.
+      These are stored in <strong>~/.local/share/applications</strong>.  The script will also create
+      desktop icons if the <strong>--desktop</strong> or the <strong>-d</strong> option is included
+      when running the script using the terminal command line, e.g.
+      <code>~/JMRI/CreateJmriApps.sh --desktop</code>.</p>
+
+      <p>For earlier versions of JMRI, the script can be downloaded from the JMRI Users Group at
+      <a href="https://groups.io/g/jmriusers/files/Script%20examples/CreateJmriApps.sh">
+      CreateJmriApps.sh</a>.
+      After downloading the script, move it to JMRI program directory and change the permissions
+      to make it executable.</p>
```

Comment:
Maybe add this command to the text?
`chmod u+x ~/JMRI/CreateJmriApps.sh`
or
`chmod +x ~/JMRI/CreateJmriApps.sh`